CHANGELOG.md

```diff
@@ -240,6 +240,10 @@ Pragmas and options
 * Benign warnings are now printed together with their warning name, to give a hint how they can be disabled
   (see [#6229](https://github.com/agda/agda/issues/6229)).
 
+* New option `--level-universe` to make `Level` inhabit its own universe `LevelUniverse`:
+  When this option is turned on, `Level` can now only depend on terms of type `Level`.
+  Note: This option is currently not compatible with cubical primitives.
```

Comment:
Does this mean that one cannot use `--level-universe` and `--cubical`/`--erased-cubical` at the same time, or that one can get internal errors, or something else?

Reply:
The option is compatible with `--cubical`/`erased-cubical`, but trying to import the builtin files will lead to an error, which will be raised by the fact that `level-universe` is coinfective but not enabled in said files. I should probably describe this more properly in the manual.

Reply:
@arthur-adjedj wrote: 
> I should probably describe this more properly in the manual.

This is still a TODO for this PR, am I right?